calibration/CTDPFB/ATAPL-67627-00005__20150510 - Copy.csv

```diff
@@ -0,0 +1,24 @@
+serial,name,value,notes
```

Comment:
remove this file from the commit

Reply:
It was supposed to be renamed to an ATCWK instrument. Has been fixed and currently undergoing CI testing